### PR TITLE
docs: update promoting content

### PR DESCRIPTION
**Pull request by jpetey75**
### Description:

Clarifying docs on promoting content since it wasn't clear promoting doesn't work in private spaces yet. 

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

**Comment by jpetey75** [comment]
@rephus when working in a preview environment, does it automatically set the upstream project? I want to add a note about working with previews to this doc but I'm not 100% sure how it works yet.

**Comment by rephus** [comment]
Yes, when you create a preview environment, (and you have a project set on the CLI) it will automatically set the upstream project and preview will initially duplicate all the content (if permissions are right IIRC)

File: docs/docs/guides/how-to-promote-content.md [hunk 1]
```diff
@@ -12,17 +12,17 @@ Promoting content enables you to copy content from one project to another. It's
 
 ## How promoting works
 
-If you are promoting a chart, from one `development project` to a `production project`, this will happen:
+If you are promoting a chart, from a `development` project to a `production` project, this will happen:
 
-- If the chart exists in both the `development project` and the `production project`, the chart in the `production` will be updated with the changes from `development`. You can always revert this chart to a previous version using [version history](./version-history)
-- If the chart is new, we will replicate this chart into the `production project`. We will also create a new space if needed.
+- If the chart exists (with the same name) in both the `development` project and the `production` project, the chart in `production` will be updated with the changes from `development`. You can always revert this chart to a previous version using [version history](./version-history)
```
Comment (rephus) [inline docs/docs/guides/how-to-promote-content.md:17]: `(with the same name)` This is not 100% right, but close. 

You can rename a chart, and still keep promoting the same chart, becuase we use `slugs` to identify chats internally. Slugs are initially created based on the chart/dashboard name, and never modified. 

I don't think there is a better way to explain what a slug is to a user, ,so we can keep it as it is.

File: docs/docs/guides/how-to-promote-content.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ style={{ display: 'block', margin: '0 auto 20px auto' }}
 
 :::info
 
-You must be a `developer` and have access to the chart/space in both the `development project` and the `upstream project`
+You must be a `developer` and have access to the chart and space in both the `development project` and the `upstream project`. The space must also be public—promoting content in private or restricted access spaces is not supported yet.
```
Comment (rephus) [inline docs/docs/guides/how-to-promote-content.md:54]: :+1:

**Review by rephus — APPROVED** [review]

**Pull request merged**